apps/formbricks-com/app/docs/contributing/gitpod/page.mdx

```diff
@@ -1,15 +1,97 @@
+- You can use either choose either VS Code Browser or  VS Code Desktop editor with the 'Standard Class' for your workspace class.
+- If you opt for the VS Code Desktop, Gitpod will prompt you to grant access to the VSCode app. Once approved, install the GitPod extension from the VSCode Marketplace and follow the prompts to authorize the integration.
+
+
+**Gitpod preparing the created Workspace**
+<Image src={GitpodPreparing} alt="Gitpod Preparing Worskpace Page" quality="100" className="rounded-lg max-w-full sm:max-w-3xl" />
+- Gitpod is preparing your workspace with all the necessary dependencies and configurations. You will see this page while Gitpod sets up your development environment.
+
+**Gitpod running the Workspace**
+<Image src={GitpodRunning} alt="Gitpod Running Workspace Page" quality="100" className="rounded-lg max-w-full sm:max-w-3xl" />
+- Once the workspace is fully prepared, voila, it enters the running state. You can start working on your project in this environment.
+
+### Ports and Services
+
+Here are the ports and corresponding URLs for the services within your Gitpod environment:
+
+- **Port 3000**:
+   - **Service**: Demo App
+   - **Description**: This port hosts the demo application of your project. You can access and interact with your application's demo by navigating to this port.
+
+- **Port 3001**:
+   - **Service**: Formbricks website
+   - **Description**: This port hosts the [Formbricks](https://formbricks.com) website, which contains documents, pricing, blogs, best practices, and concierge service.
+
+- **Port 3002**:
+   - **Service**: Formbricks In-product Survey Demo App
+   - **Description**: This app helps you test your in-app surveys. You can create and test user actions, create and update user attributes, etc.
+
+- **Port 5432**:
+   - **Service**: PostgreSQL Database Server
+   - **Description**: The PostgreSQL DB is hosted on this port.
+
+- **Port 1025**:
+   - **Service**: SMPT server
+   - **Description**: SMTP Server for sending and receiving email messages. This server is responsible for handling email communication.
+
+- **Port 8025**:
+   - **Service**: Mailhog
+
+### Accessing port URLs
+  1. **Direct URL Composition**:
+   - You can access the dedicated port URL by pre-pending the port number to the workspace URL.
+   - For example, if you want to access port 3000, you can use the URL format: `3000-yourworkspace.ws-eu45.gitpod.io`.
+
+  2. **Using `gp url` Command**:
+   - Gitpod provides a convenient command, `gp url`, to quickly retrieve the URL for a specific port.
+   - Simply use the command followed by the desired port number. For example, to get the URL for port 3000, run: `gp url 3000`.
+
+  3. **Listing All Open Port URLs**:
+   - If you prefer to see a list of all open port URLs at once, you can use the `gp ports list` command.
+   - Running this command will display a list of ports along with their corresponding URLs.
```

Comment:
Can you add a note here for the `gp` command since you're referring to it here and somebody would not know what it means / where to get it from. So maybe a note/hyperlink here that takes them to the downloadable/actionable page of gp cli.

Reply:
sure